video/out/opengl/egl_helpers.c

```diff
@@ -100,8 +101,34 @@ static void *mpegl_get_proc_address(void *ctx, const char *name)
     return p;
 }
 
+struct egl_opts {
+    int config_id;
+    int output_format;
+};
+
+#define MAKE_FORMAT(r, g, b, a) ((r) << 24 | (g) << 16 | (b) << 8 | (a))
```

Comment:
suggestion to make this clearer (IMO):
use `((r) << 18 | (g) << 12 | (b) << 6 | (a))` and then you can use `0x1000000` to indicate float to instead of overloading the sign bit

Reply:
modified